### PR TITLE
Update Docker setup instructions

### PR DESCRIPTION
**Pull request by jerivas**
## Cute animal pic
_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

## Description

Tweaked documentation and example env file for user of Docker and specifically devcontainers on VS Code

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

## Show me
_Provide screenshots/animated gifs/videos if necessary._

### CODE REVIEW

File: .env.docker.example [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # DB settings
+# Ensure these match with docker-compose.yml
 POSTGRES_DB=metadeploy
 POSTGRES_USER=postgres
-POSTGRES_PASSWORD=
-# General settings
+POSTGRES_PASSWORD=sample_db_password
```
Comment (davisagli) [inline .env.docker.example:5]: Salesforce security team requires us to not include passwords in the repository, even if it's a sample password for a local environment.

File: docs/running_docker.rst [hunk 2]
```diff
@@ -75,22 +75,28 @@ POSTGRES_DB:
...
+CONNECTED_APP_CALLBACK_URL:
+    Callback url of the connected Salesforce app. The variable has already been set to the proper value for projects running under localhost.
+
+.. note::
+
+    If you're an OddBird, you can find the following values in the shared Keybase team folder -- ``metadeploy/env``
+
 CONNECTED_APP_CLIENT_ID:
-    This represents the consumer key of the connected app that MetaDeploy will use for authenticating to any persistent org.
+    Consumer key of the connected app that MetaDeploy will use for authenticating to any persistent org. If you're and OddBird you can retrieve this value
```
Comment (davisagli) [inline docs/running_docker.rst:96]: This looks like an incomplete sentence that's repeating line 93?

**Review by davisagli — CHANGES_REQUESTED** [review]
Thanks for taking the time to make the docs better. We can't include the db password here, but the rest looks good.

**Comment by jerivas** [comment]
Thanks for flagging @davisagli. Should be fixed now.

**Review by davisagli — COMMENTED** [review]
Looks good to me now. Needs to have main merged in again -- you should probably coordinate with @jgerigmeyer on that since he's working on merging some upstream changes there.

**Review by davisagli — APPROVED** [review]

**Review by jgerigmeyer — APPROVED** [review]

**Pull request merged**